src/commands/zone/check.ts

```diff
@@ -0,0 +1,84 @@
+import inquirer from 'inquirer';
+import Command, { IConfig } from '../../base.js';
+import { Flags } from '@oclif/core';
+import { createDebugLogger } from '../../utils/output.js';
+import { ux } from '@oclif/core';
+
+export default class Check extends Command {
+  static description = 'Check a zone.';
+
+  static baseURL = 'https://dns-service.iran.liara.ir';
```

Comment:
Please move this to the `constants.ts` file.

Reply:
Right. We have other things that should be moved to its place like Interfaces. But for now, to keep avoiding merge conflict, I defined everything in a single file.

Reply:
OK. Don't forget to move them to the constants file after the merge.